config/webpack.config.lib.js

```diff
@@ -27,11 +27,6 @@ module.exports = {
     libraryTarget: 'umd',
   },
 
-  // Enable sourcemaps for debugging webpack's output.
-  // If it is inline, it will break CSS sourcemaps because of
-  // `extract-text-webpack-plugin`
-  devtool: 'source-map',
```

Comment:
I think having totally no sourcemap at all will make it harder to users lib-voyager to report bug?